<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZirconiumPass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1203</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionCreate_New"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_Zirconiumpass"/>
   </widget>
   <widget class="QMenu" name="menuEntry">
    <property name="title">
     <string>&amp;Entry</string>
    </property>
    <addaction name="actionCreateNewEntry"/>
    <addaction name="actionEdit_Entry"/>
    <addaction name="actionDelete_Entry"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEntry"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Existing</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCreate_New">
   <property name="text">
    <string>Create &amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionQuit_Zirconiumpass">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit Zirconiumpass</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCreateNewEntry">
   <property name="text">
    <string>Create N&amp;ew</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionEdit_Entry">
   <property name="text">
    <string>Edit selected Ent&amp;ry</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionDelete_Entry">
   <property name="text">
    <string>&amp;Delete selected Entry</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit_Zirconiumpass</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDatabaseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCreate_New</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>createNewDatabaseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeDatabaseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCreateNewEntry</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>createNewEntryClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveDatabaseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit_Entry</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>editEntryClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete_Entry</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteEntryClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openDatabaseClicked()</slot>
  <slot>closeDatabaseClicked()</slot>
  <slot>createNewDatabaseClicked()</slot>
  <slot>createNewEntryClicked()</slot>
  <slot>saveDatabaseClicked()</slot>
  <slot>editEntryClicked()</slot>
  <slot>deleteEntryClicked()</slot>
 </slots>
</ui>
